### PR TITLE
Make features and benefits editable

### PR DESCRIPTION
**Pull request by quis**
The rules around editing features and benefits are:
- Maximum of 10 items per list
- Each item can't be more than 10 words in length
- If there is a gap between submitted items, it should be filtered out

### CODE REVIEW

**Comment by quis** [comment]
Javascript parts aren't working, closing temporarily…

**Comment by quis** [comment]
Edit: going to raise a separate pull request for adding Javascript enhancement. Review away.

File: app/main/views.py [hunk 1]
```diff
@@ -106,9 +106,12 @@ def update(service_id, section):
 
     # Turn responses which have multiple parts into lists
     for item in request.form.items():
```
Comment (allait) [inline app/main/views.py]: `for key in request.form:`?

**Comment by allait** [comment]
Looks great, `request.form` is a nitpick, happy to merge as is :+1:

File: app/main/views.py (hunk 1 above)
Comment (quis) [inline app/main/views.py]: @allait a79948a

**Pull request merged**